charmcraft.yaml

```diff
@@ -24,6 +24,21 @@ bases:
 containers:
   gocert:
     resource: gocert-image
+    mounts:
+    - storage: config
+      location: /var/lib/gocert/config
```

Comment:
I think @gruyaume's comment was only for the DB, not the configuration. I think the config makes more sense in `/etc`

Reply:
changed it back